---
content_type: page
description: This section provides the citations and links to the readings assigned
  for each session of the course.
learning_resource_types:
- Readings
ocw_type: CourseSection
title: Readings
uid: 1f3d1984-e0db-6e25-65c7-313934369c19
video_metadata:
  youtube_id: null
---

This textbook is recommended for the course:  
Zvelebil, Marketa J., and Jeremy O. Baum. _Understanding Bioinformatics_. Garland Science, 2007. ISBN: 9780815340249. \[Preview with [Google books](http://books.google.com/books?id=dGayL_tdnBMC&printsec=frontcover)\]

The instructors have also selected various texts as particularly useful in specific areas, if you are looking for more information. See the {{% resource_link fa95efc3-d3fe-cba9-9745-73504676f128 "textbook section on the syllabus" "#textbook" %}}.

{{< tableopen >}}
{{< theadopen >}}
{{< tropen >}}
{{< thopen >}}
LEC #
{{< thclose >}}
{{< thopen >}}
TOPICS
{{< thclose >}}
{{< thopen >}}
READINGS
{{< thclose >}}

{{< trclose >}}

{{< theadclose >}}
{{< tropen >}}
{{< tdopen >}}
1
{{< tdclose >}}
{{< tdopen >}}
Course Introduction; Overview
{{< tdclose >}}
{{< tdopen >}}
No readings for this lecture.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
2
{{< tdclose >}}
{{< tdopen >}}
Local Alignment; Statistics
{{< tdclose >}}
{{< tdopen >}}


National Center for Biotechnology Information. "[The Statistics of Sequence Similarity Scores](http://www.ncbi.nlm.nih.gov/BLAST/tutorial/Altschul-1.html)." BLAST Tutorial.

Metzker, Michael L. "[Sequencing Technologies—The Next Generation](http://dx.doi.org/10.1038/nrg2626)." _Nature Reviews Genetics_ 11, no. 1 (2010): 31–46.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
3
{{< tdclose >}}
{{< tdopen >}}
Global Alignment of Protein Statistics
{{< tdclose >}}
{{< tdopen >}}
No readings for this lecture.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
4
{{< tdclose >}}
{{< tdopen >}}
Comparative Genomics
{{< tdclose >}}
{{< tdopen >}}


Sabeti, P. C., S. F. Schaffner, et al. "[Positive Natural Selection in the Human Lineage](http://dx.doi.org/10.1126/science.1124309)." _Science_ 312, no. 5780 (2006): 1614–20.  
Read the first three pages.

Bejerano, Gill, Michael Pheasant, et al. "[Ultraconserved Elements in the Human Genome](http://dx.doi.org/10.1126/science.1098119)." _Science_ 304, no. 5675 (2004): 1321–5.

Pennacchio, Len A., Nadav Ahituv, et al. "[In Vivo Enhancer Analysis of Human Conserved Non–coding Sequences](http://dx.doi.org/10.1038/nature05295)." _Nature_ 444, no. 7118 (2006): 499–502.

Visel, Axel, Shyam Prabhakar, et al. "[Ultraconservation Identifies a Small Subset of Extremely Constrained Developmental Enhancers](http://dx.doi.org/10.1038/ng.2007.55)." _Nature Genetics_ 40, no. 2 (2008): 158–60.

Bejerano, Gill, Craig B. Lowe, et al. "[A Distal Enhancer and an Ultraconserved Exon are Derived from a Novel Retroposon](http://dx.doi.org/10.1038/nature04696)." _Nature_ 441, no. 7089 (2006): 87–90.

Lareau, Liana F., Maki Inada, et al. "[Unproductive Splicing of SR Genes Associated with Highly Conserved and Ultraconserved DNA Elements](http://dx.doi.org/10.1038/nature05676)." _Nature_ 446, no. 7138 (2007): 926–9.

Lewis, Benjamin P., I–hung Shih, et al. "[Prediction of Mammalian MicroRNA Targets](http://dx.doi.org/10.1016/S0092-8674(03)01018-3)." _Cell_ 115, no. 7 (2003): 787–98.

Lewis, Benjamin P., Christopher B. Burge, et al. "[Conserved Seed Pairing, Often Flanked by Adenosines, Indicates that Thousands of Human Genes are MicroRNA Targets](http://dx.doi.org/10.1016/j.cell.2004.12.035)." _Cell_ 120, no. 1 (2005): 15–20.

Kheradpour, Pouya, Alexander Stark, et al. "[Reliable Prediction of Regulator Targets Using 12 Drosophila Genomes](http://dx.doi.org/10.1101/gr.7090407)." _Genome Research_ 17, no. 12 (2007): 1919–31.

Friedman, Robin C., Kyle Kai–How Farh, et al. "[Most Mammalian mRNAs are Conserved Targets of MicroRNAs](http://dx.doi.org/10.1101/gr.082701.108)." _Genome Research_ 19, no. 1 (2009): 92–105.

Graveley, Brenton R. "[Mutually Exclusive Splicing of the Insect Dscam Pre–mRNA Directed by Competing Intronic RNA Secondary Structures](http://dx.doi.org/10.1016/j.cell.2005.07.028)." _Cell_ 123, no. 1 (2005): 65–73.

Jansen, Ruud, Jan Embden, et al. "[Identification of Genes that are Associated with DNA Repeats in Prokaryotes](http://dx.doi.org/10.1046/j.1365-2958.2002.02839.x)." _Molecular Microbiology_ 43, no. 6 (2002): 1565–75.

Bolotin, Alexander, Benoit Quinquis, et al. "[Clustered Regularly Interspaced Short Palindrome Repeats (CRISPRs) have Spacers of Extrachromosomal Origin](http://dx.doi.org/10.1099/mic.0.28048-0)." _Microbiology_ 151, no. 8 (2005): 2551–61.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
5
{{< tdclose >}}
{{< tdopen >}}
Read Alignment
{{< tdclose >}}
{{< tdopen >}}


Langmead, Ben, Cole Trapnell, et al. "[Ultrafast and Memory–efficient Alignment of Short DNA Sequences to the Human Genome](http://dx.doi.org/10.1186/gb-2009-10-3-r25)." _Genome Biology_ 10, no. 3 (2009): R25.

Li, Heng, and Richard Durbin. "[Fast and Accurate Short Read Alignment with Burrows–wheeler Transform](http://dx.doi.org/10.1093/bioinformatics/btp324)." _Bioinformatics_ 25, no. 14 (2009): 1754–60.

Trapnell, Cole, and Steven L. Salzberg. "[How to Map Billions of Short Reads onto Genomes](http://dx.doi.org/10.1038/nbt0509-455)." _Nature Biotechnology_ 27, no. 5 (2009): 455.

[Burrows–Wheeler Aligner](http://bio-bwa.sourceforge.net/)

[Bowtie](http://bowtie-bio.sourceforge.net/index.shtml): An ultrafast memory–efficient short read aligner


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
6
{{< tdclose >}}
{{< tdopen >}}
Genome Assembly
{{< tdclose >}}
{{< tdopen >}}


Simpson, Jared T., and Richard Durbin. "[Efficient De Novo Assembly of Large Genomes Using Compressed Data Structures](http://dx.doi.org/10.1101/gr.126953.111)." _Genome Research_ 22, no. 3 (2012): 549–56.

Zerbino, Daniel R., and Ewan Birney. "[Velvet: Algorithms for De Novo Short Read Assembly Using De Bruijn Graphs](http://dx.doi.org/10.1101/gr.074492.107)." _Genome Research_ 18, no. 5 (2008): 821–9.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
7
{{< tdclose >}}
{{< tdopen >}}
ChIP-seq / IDR
{{< tdclose >}}
{{< tdopen >}}


Guo, Yuchun, Georgios Papachristoudis, et al. "[Discovering Homotypic Binding Events at High Spatial Resolution](http://dx.doi.org/10.1093/bioinformatics/btq590)." _Bioinformatics_ 26, no. 24 (2010): 3028–34.

Li, Qunhua, James B. Brown, et al. "[Measuring Reproducibility of High–throughput Experiments](http://dx.doi.org/10.1214/11-AOAS466)." _The Annals of Applied Statistics_ 5, no. 3 (2011): 1752–79.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
8
{{< tdclose >}}
{{< tdopen >}}
RNA–seq Analysis
{{< tdclose >}}
{{< tdopen >}}


Trapnell, Cole, Brian A. Williams, et al. "[Transcript Assembly and Quantification by RNA–seq Reveals Unannotated Transcripts and Isoform Switching during Cell Differentiation](http://dx.doi.org/10.1038/nbt.1621)." _Nature Biotechnology_ 28, no. 5 (2010): 511–5.

Anders, Simon, and Wolfgang Huber. "[Differential Expression Analysis for Sequence Count Data](http://dx.doi.org/10.1186/gb-2010-11-10-r106)." _Genome Biology_ 11, no. 10 (2010): R106.  
Licensed under CC–BY.

Wang, Zhong, Mark Gerstein, et al. "[RNA–Seq: a Revolutionary Tool for Transcriptomics](http://dx.doi.org/10.1038/nrg2484)." _Nature Reviews Genetics_ 10, no. 1 (2009): 57–63.

Shalek, Alex K., Rahul Satija, et al. "[Single–cell Transcriptomics Reveals Bimodality in Expression and Splicing in Immune Cells](http://dx.doi.org/10.1038/nature12172)." _Nature_ 498 (2013): 236–40.

Smith, Lindsay I. ["A Tutorial on Principal Components Analysis." (PDF)](http://www.cs.otago.ac.nz/cosc453/student_tutorials/principal_components.pdf) February 26, 2002.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
9
{{< tdclose >}}
{{< tdopen >}}
Modeling and Discovery of Sequence Motifs (Gibbs Sampler, Alternatives)
{{< tdclose >}}
{{< tdopen >}}


D'haeseleer, Patrik. "[What are DNA Sequence Motifs?](http://dx.doi.org/10.1038/nbt0406-423)" _Nature Biotechnology_ 24, no. 4 (2006): 423–25.

———. "[How does DNA Sequence Motif Discovery Work?](http://dx.doi.org/10.1038/nbt0806-959)" _Nature Biotechnology_ 24, no. 8 (2006): 959–61.

Eddy, Sean R. "[What is Bayesian Statistics?](http://dx.doi.org/10.1038/nbt0904-1177)" _Nature Biotechnology_ 22, no. 9 (2004): 1177–8.

Bailey, Timothy L., and Charles Elkan. "[Unsupervised Learning of Multiple Motifs in Biopolymers Using Expectation Maximization](http://dx.doi.org/10.1007/BF00993379)." _Machine Learning_ 21, no. 1–2 (1995): 51–80.

Lawrence, Charles E., Stephen F. Altschul, et al. "[Detecting Subtle Sequence Signals: A Gibbs Sampling Strategy for Multiple Alignment](http://dx.doi.org/10.1126/science.8211139)." _Science_ 262, no. 5131 (1993): 208–14.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
10
{{< tdclose >}}
{{< tdopen >}}
Markov and Hidden Markov Models
{{< tdclose >}}
{{< tdopen >}}


Eddy, Sean R. "[What is a Hidden Markov Model?](http://dx.doi.org/10.1038/nbt1004-1315)" _Nature Biotechnology_ 22, no. 10 (2004): 1315–6.

Rabiner, Lawrence. "[A Tutorial on Hidden Markov Models and Selected Applications in Speech Recognition](http://dx.doi.org/10.1109/5.18626)." _Proceedings of the IEEE_ 77, no. 2 (1989): 257–86.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
11
{{< tdclose >}}
{{< tdopen >}}
RNA Secondary Structure Prediction
{{< tdclose >}}
{{< tdopen >}}
Eddy, Sean R. "[How do RNA Folding Algorithms Work?](http://dx.doi.org/10.1038/nbt1104-1457)" _Nature Biotechnology_ 22, no. 11 (2004): 1457–8.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
12
{{< tdclose >}}
{{< tdopen >}}
Introduction to Protein Structure
{{< tdclose >}}
{{< tdopen >}}


Scheeff, Eric D., and J. Lynn Fink. "Fundamentals of Protein Structure." In _Structural Bioinformatics_. Edited by Philip E. Bourne and Helge Weissig. Wiley–Liss, 2003, pp. 15–39. \[Preview with [Google Books](http://books.google.com/books?id=4H_ai7ivRIcC&pg=PA15#v=onepage)\]


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
13
{{< tdclose >}}
{{< tdopen >}}
Predicting Protein Structure
{{< tdclose >}}
{{< tdopen >}}
Moretti, Rocco, Sarel J. Fleishman, et al. "[Community‐wide Evaluation of Methods for Predicting the Effect of Mutations on Protein–protein Interactions](http://dx.doi.org/10.1002/prot.24356)." _Proteins: Structure, Function, and Bioinformatics_ 81, no. 11 (2013): 1980–7.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
14
{{< tdclose >}}
{{< tdopen >}}
Predicting Interactions
{{< tdclose >}}
{{< tdopen >}}


Tuncbag, Nurcan, Attila Gursoy, et al. "[Predicting Protein–protein Interactions on a Proteome Scale by Matching Evolutionary and Structural Similarities at Interfaces Using PRISM](http://dx.doi.org/10.1038/nprot.2011.367)." _Nature Protocols_ 6, no. 9 (2011): 1341–54.

Zhang, Qiangfeng Cliff, Donald Petrey, et al. "[Structure–based Prediction of Protein–protein Interactions on a Genome–wide Scale](http://dx.doi.org/10.1038/nature11503)." _Nature_ 490, no. 7421 (2012): 556–60.

Jansen, Ronald, Haiyuan Yu, et al. "[A Bayesian Networks Approach for Predicting Protein–protein Interactions from Genomic Data](http://dx.doi.org/10.1126/science.1087361)." _Science_ 302, no. 5644 (2003): 449–53.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
15
{{< tdclose >}}
{{< tdopen >}}
Gene Regulatory Networks
{{< tdclose >}}
{{< tdopen >}}
Marbach, Daniel, James C. Costello, et al. "[Wisdom of Crowds for Robust Gene Network Inference](http://dx.doi.org/10.1038/nmeth.2016)." _Nature Methods_ 9, no. 8 (2012): 796–804.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
16
{{< tdclose >}}
{{< tdopen >}}
Protein Interaction Networks
{{< tdclose >}}
{{< tdopen >}}
No readings for this lecture.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
17
{{< tdclose >}}
{{< tdopen >}}
Logic Modeling of Cell Signaling Networks. Guest Lecture: [Doug Lauffenburger](http://web.mit.edu/dallab/index.html)
{{< tdclose >}}
{{< tdopen >}}


Morris, Melody K., Julio Saez–Rodriguez, et al. "[Logic–based Models for the Analysis of Cell Signaling Networks](http://dx.doi.org/10.1021/bi902202q)." _Biochemistry_ 49, no. 15 (2010): 3216–24.

Saez‐Rodriguez, Julio, Leonidas G. Alexopoulos, et al . "[Discrete Logic Modelling as a Means to Link Protein Signalling Networks with Functional Analysis of Mammalian Signal Transduction](http://dx.doi.org/10.1038/msb.2009.87)." _Molecular Systems Biology_ 5, no. 1 (2009): 331.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
18
{{< tdclose >}}
{{< tdopen >}}
Analysis of Chromatin Structure
{{< tdclose >}}
{{< tdopen >}}


Hoffman, Michael M., Orion J. Buske, et al. "[Unsupervised Pattern Discovery in Human Chromatin Structure through Genomic Segmentation](http://dx.doi.org/10.1038/nmeth.1937)." _Nature Methods_ 9, no. 5 (2012): 473–6.

Zhou, Vicky W., Alon Goren, et al. "[Charting Histone Modifications and the Functional Organization of Mammalian Genomes](http://dx.doi.org/10.1038/nrg2905)." _Nature Reviews Genetics_ 12, no. 1 (2010): 7–18.

Sherwood, Richard I., Tatsunori Hashimoto, et al. "[Discovery of Directional and Nondirectional Pioneer Transcription Factors by Modeling DNase Profile Magnitude and Shape](http://dx.doi.org/10.1038/nbt.2798)." _Nature Biotechnology_ 32, no. 2 (2014): 171–8.

Dostie, Josée, and Job Dekker. "[Mapping Networks of Physical Interactions between Genomic Elements Using 5C Technology](http://dx.doi.org/10.1038/nprot.2007.116)." _Nature Protocols_ 2, no. 4 (2007): 988–1002.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
19
{{< tdclose >}}
{{< tdopen >}}
Discovering Quantitative Trait Loci (QTLs)
{{< tdclose >}}
{{< tdopen >}}


Bloom, Joshua S., Ian M. Ehrenreich, et al. "[Finding the Sources of Missing Heritability in a Yeast Cross](http://dx.doi.org/10.1038/nature11867)." _Nature_ 494, no. 7436 (2013): 234–7.

Broman, Karl W., and Saunak Sen. "Single–QTL Analysis." Chapter 4 in _A Guide to QTL Mapping with R/qtl_. Springer, 2009. ISBN: 9780387921242. \[Preview with [Google Books](http://books.google.com/books?id=tPoXT_dCguQC&pg=PA75#v=onepage)\]


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
20
{{< tdclose >}}
{{< tdopen >}}
Genome Wide Associate Studies
{{< tdclose >}}
{{< tdopen >}}


Li, Heng. "[A Statistical Framework for SNP Calling, Mutation Discovery, Association Mapping and Population Genetical Parameter Estimation from Sequencing Data](http://dx.doi.org/10.1093/bioinformatics/btr509)." _Bioinformatics_ 27, no. 21 (2011): 2987–93.

Roberts, Nicholas J., Joshua T. Vogelstein, et al. "[The Predictive Capacity of Personal Genome Sequencing](http://dx.doi.org/10.1126/scitranslmed.3003380)." _Science Translational Medicine_ 4, no. 133 (2012): 133ra58.

1000 Genomes. "[Variant Call Format](http://www.1000genomes.org/wiki/analysis/variant-call-format/vcf-variant-call-format-version-42)."

Goldstein, David B., Andrew Allen, et al. "[Sequencing Studies in Human Genetics: Design and Interpretation](http://dx.doi.org/10.1038/nrg3455)." _Nature Reviews Genetics_ 14, no. 7 (2013): 460–70.


{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
21
{{< tdclose >}}
{{< tdopen >}}
Synthetic Biology: From Parts to Modules to Therapeutic Systems. Guest Lecture: [Ron Weiss](http://groups.csail.mit.edu/synbio/)
{{< tdclose >}}
{{< tdopen >}}
No readings for this lecture.
{{< tdclose >}}

{{< trclose >}}
{{< tropen >}}
{{< tdopen >}}
22
{{< tdclose >}}
{{< tdopen >}}
Causality, Natural Computing, and Engineering Genomes. Guest Lecture: [George Church](http://arep.med.harvard.edu/gmc/)
{{< tdclose >}}
{{< tdopen >}}
No readings for this lecture.
{{< tdclose >}}

{{< trclose >}}

{{< tableclose >}}